doc/api/tls.md

```diff
@@ -445,6 +445,47 @@ The first three are enabled by default. The two `CCM`-based suites are supported
 by TLSv1.3 because they may be more performant on constrained systems, but they
 are not enabled by default since they offer less security.
 
+## OpenSSL Security Level
+
+The OpenSSL library enforces security levels to control the minimum acceptable
+level of security for cryptographic operations. OpenSSL's security levels range
+from 0 to 5, with each level imposing stricter security requirements. The default
+security level is 1, which is generally suitable for most modern applications.
+However, some legacy features and protocols, such as TLSv1, require a lower
+security level (`SECLEVEL=0`) to function properly. For more detailed information,
+please refer to the [OpenSSL documentation on security levels][].
+
+### Setting Security Levels
+
+To adjust the security level in your Node.js application, you can include `@SECLEVEL=X`
+within a cipher string, where `X` is the desired security level. For example,
+to set the security level to 0 while using the default OpenSSL cipher list, you could use:
+
+```js
+const tls = require('tls');
+const port = 443;
+
+tls.createServer({ciphers: 'DEFAULT@SECLEVEL=0', minVersion: 'TLSv1'}, function (socket) {
+  console.log('it works!', socket.getProtocol());
```

Comment:
nit: Maybe "it works!" isn't the best log, as it would be assumed to work, maybe "The protocol is: " would be better?

Reply:
+1

Maybe `Client connected with protocol:` or similar, just to make it clear that this callback running does mean a connection has really worked.